scanners/reactnative/expo.go

```diff
@@ -27,36 +22,30 @@ const (
 )
 
 // expoOptions implements ScannerInterface.Options function for Expo based React Native projects.
-func (scanner *Scanner) expoOptions() (models.OptionNode, models.Warnings, error) {
+func (scanner *Scanner) expoOptions() (models.OptionNode, models.Warnings) {
```

Comment:
NOTE: could warning be removed as well from the return values?

Reply:
Removed.